docs/sourcemaps.md

```diff
@@ -0,0 +1,86 @@
+---
+id: sourcemaps
+title: Source Maps
+---
+
+Source maps help map a minified file back to an un-packaged state. This is common when investigating issues from release builds.
+
+This helps if you have a stacktrace like
+
+```text
+TypeError: Cannot read property 'data' of undefined
+  at anonymous(app:///index.android.bundle:1:4277021)
+  at call(native)
+  at p(app:///index.android.bundle:1:227785)
+```
+
+The power of source maps allow tracing that line and column number back to the original source stacktrace.
```

Comment:
```suggestion
With source maps generated, a stacktrace will include path, file name, line and column numbers of the original source file:
```

And one more. 😉

Reply:
Agree, "column" was in the original sentence before I have posted suggestion, thought about removing it, but I left it at the end, so yeah, let's drop it.

Reply:
sounds good. Take a look now and let me know what you think.

Reply:
LGTM, thank you for the quick review updates! 🙏